File: Exercicio02/NotaAluno.ts
class NotaAluno{
    public static nota: number = 0;
    public static soma: number = 0;
    public static contNota: number = 1;
    public static main(): void{
        this.soma = 0;
        this.contNota = 0;
        while(this.contNota < 3){
            let aux = prompt("Digite a nota ") + "";
            this.nota = parseFloat(aux);
            while(this.nota < 0){
                alert("Nota Inválida");
                this.nota = parseFloat(prompt('Digite a nota ')+"");
            }
            this.soma += this.nota;
            this.contNota++;
        }
        alert(`O aluno teve um total de: ${this.soma}`);
    }
    
}
NotaAluno.main();

File: Exercicio02/NotaAluno.java
package exercicios;

import java.util.Scanner;

public class NotaAluno {

	public static void main(String[] args) {

		Scanner leia = new Scanner(System.in);
		
		float nota, soma=0;
		int contNota=1;
		
		
		soma = 0;
		contNota = 0;
		while(contNota<3){
			System.out.print("Digite a nota "+(contNota+1)+": ");
			nota = leia.nextFloat();
			while (nota<0) {
				System.out.println("NOTA INVALIDA!!!");
				System.out.print("Digite a nota "+(contNota+1)+": ");
				nota = leia.nextFloat();
				
			}
			soma += nota;
			contNota++;
			
		};
		System.out.println("O aluno teve um total de: "+soma);

		leia.close();

	}

}
